### PR TITLE
Update wasm getting started instructions

### PR DESCRIPTION
**Pull request by mit-mit**
_Description of what this PR is changing or adding, and why:_

Update Wasm getting started instructions to:
- Simplify a bit
- Rely on the standard template to ensure they are compatible
- Align with bootstrap changes landed in https://github.com/flutter/flutter/pull/144434

## Presubmit checklist

- [ ] This PR doesn’t contain automatically generated corrections (Grammarly or similar).
- [ ] This PR follows the [Google Developer Documentation Style Guidelines](https://developers.google.com/style) — for example, it doesn’t use _i.e._ or _e.g._, and it avoids _I_ and _we_ (first person).
- [ ] This PR uses [semantic line breaks](https://github.com/dart-lang/site-shared/blob/main/doc/writing-for-dart-and-flutter-websites.md#semantic-line-breaks) of 80 characters or fewer.

### CODE REVIEW

**Comment by MaryaBelanger** [comment]
@mit-mit this might be a duplicate of https://github.com/flutter/website/pull/10249

**Comment by mit-mit** [comment]
Yeah, somewhat duplicative it seems!

We can't recommend beta just yet, as the build change (see my link above) just landed last night.

In either case I'd love it if either of the PRs can be landed today, as @kevmoo is doing his talk tomorrow.

**Comment by mit-mit** [comment]
Oh, the other PR landed. But @kevmoo doesn't that mean the instructions don't work as they assume code that is only in main but the instructions say to use beta?

**Comment by mit-mit** [comment]
Updated this after the changes from the other PR.

**Review by sfshaza2 — APPROVED** [review]
lgtm

File: src/platform-integration/web/wasm.md [hunk 1]
```diff
@@ -154,12 +123,16 @@ Server started on port 8080
...
+When the app is loaded, it will detect if the browser supports Wasm,
+and fall back to the JavaScript version if it doesn't.
+
 As of {{page.last-update}},
 [only **Chromium-based browsers**](#chrome-119-or-later)
 (Version 119 or later) are able to run Flutter/Wasm content. 
 
-If your configured browser meets the requirements, open
-[`localhost:8080`](http://localhost:8080) in the browser to view the app.
+Open [`localhost:8080`](http://localhost:8080)
```
Comment (sfshaza2) [inline src/platform-integration/web/wasm.md:134]: I suspect that the broken build is due to this link.

Comment (mit-mit) [reply src/platform-integration/web/wasm.md:134]: That link was introduced by a different PR, so should probably be fixed separately from this.

**Comment by mit-mit** [comment]
@sfshaza2 this was ready to be merged (sorry, I should have marked it draft). Can you please revert?

**Pull request merged**